libtelemetry/src/main/java/com/mapbox/android/telemetry/CertificateBlacklist.java

```diff
@@ -100,7 +101,9 @@ private static boolean isValidContent(String data) {
     JsonArray jsonArray;
     try {
       JsonObject responseJson = gson.fromJson(data, JsonObject.class);
-      jsonArray = responseJson.getAsJsonArray("RevokedCertKeys");
+      JsonElement jsonElement = responseJson.get("RevokedCertKeys");
```

Comment:
does `responseJson.getAsJsonArray("RevokedCertKeys");` returns `null` if object is not array? so why do we need additional helper on line 140?

Reply:
During testing, this did not hold true. Test failed when writing check with `IllegalStateException`. Wrote boolean check to correct that issue. Just dug into gson documentation to confirm and that is what it says. It did enlighten me to a method that already existed for this functionality though, so will replace with new method.

https://www.javadoc.io/doc/com.google.code.gson/gson/2.8.5